### DISCUSSION TITLE
Google Pixel3 Android12 requires Always-on VPN set to on

### DISCUSSION THREAD

**Discussion by vitya76** [strongswan/strongswan#934] [Q&A]
Hello,
Seems like on my Google Pixel device on Android 12 if Always-on VPN set to OFF Chrome ignores VPN and goes to the internet directly. No matter the setting "All applications use the VPN" or "Only selected applications use VPN with Chrome checked".
When always-on VPN set to ON everything works as expected.
Thank you for your help.

**Comment by vitya76** [comment]
Update: It actually needs "Block all connections without VPN" setting to be ON

**Reply by tobiasbrunner** [reply]
You might want to report this to Google.
